packages/forms/stories/static/stringCompletionItems.json

```diff
@@ -0,0 +1,18 @@
+[{
+	"title": "A",
```

Comment:
Yes for sure. This is a first implementation to do quick tests on the integration.
It'll be improved later. The widget API isn't fixed.